.github/workflows/release.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Build and package gem artifact
         run: |
           gem build opensearch-aws-sigv4.gemspec
-          mkdir dist && mv opensearch-ruby-*.gem dist/
+          mkdir dist && mv *.gem dist/
```

Comment:
May be instead of `*`, we can do `opensearch-*.gem`?

Reply:
I dont think it matters because our release pipeline only packages one gem at a time even when we had multiple gems in one repo. I'm just trying to avoid a copy-paste error like this in the future :P